cordapp/src/main/kotlin/com.luxoft.blockchainlab.corda.hyperledger.indy/flow/IssueClaimFlow.kt

```diff
@@ -69,8 +81,11 @@ object IssueClaimFlow {
 
                 val newClaimCmd = Command(newClaimData, newClaimSigners)
 
+                val claimDefCmdType = ClaimMetadataChecker.Command.Use()
+                val claimDefCmd = Command(claimDefCmdType, newClaimSigners)
+
                 val trxBuilder = TransactionBuilder(whoIsNotary())
-                        .withItems(newClaimOut, newClaimCmd)
+                        .withItems(newClaimOut, newClaimCmd, credDefState, claimDefCmd)
```

Comment:
CredentialDefinition need to be consumed and produced new one. 
Current number of credential should be increased and checked in the contract.